CosmoSpaceShip/CosmoSpaceShip/Src/Game.cpp

```diff
@@ -347,6 +360,45 @@ void Game::clean()
 	SDL_Quit();
 }
 
+void Game::CreateMenu()
+{
+	m_gameMenu = new Menu();
+}
+
+void Game::prepareGameMenu()
+{
+	SDL_Color white = { 255,255,255,255 };
+	playLabel.addComponent<UILabel>(m_playWidth / 2 - 48, m_playHeight / 2 - 200,      "play", "courier", white);
+	exitLabel.addComponent<UILabel>(m_playWidth / 2 - 48, m_playHeight / 2 - 200 + 48, "exit", "courier", white);
+	//playLabel.addComponent<UILabel>(m_gameMenu->getLabelbyKey("play")); TBD: find out why it is not working
+}
+
+void Game::updateGameMenu()
+{
+	//update all components to the current frame state
+	manager.refresh();
+	manager.update();
+
+	if (gameMenuCooldown != 0)
+		--gameMenuCooldown;
+
+	SDL_Color red   = { 255,0,0,255 };
+	SDL_Color white = { 255,255,255,255 };
+
+	switch (m_gameMenu->getCurrentButton())
+	{
+	case MenuButtons::Play: 
+		playLabel.getComponent<UILabel>().SetLabelColor(red);
+		exitLabel.getComponent<UILabel>().SetLabelColor(white);
+		break;
+	case MenuButtons::Exit:
+		exitLabel.getComponent<UILabel>().SetLabelColor(red);
+		playLabel.getComponent<UILabel>().SetLabelColor(white);
+	default:
+		break;
+	}
```

Comment:
better to use for cycle for every button and set its color to red if it is a current button and white if it is not

Reply:
Will fix with patches